.docker/keycloak/docker-compose-keycloak-dev.yml

```
@@ -0,0 +1,46 @@
+    environment:
+      POSTGRES_DB: ${KC_POSTGRES_DB}
+      POSTGRES_USER: ${KC_POSTGRES_USER}
+      POSTGRES_PASSWORD: ${KC_POSTGRES_PASSWORD}
+    ports:
+      - "5433:5432"
+    volumes:
+      - kamann_keycloak_db_dev_volume:/var/lib/postgresql/data
+    networks:
+      - kamann_network
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U ${KC_POSTGRES_USER} -d ${KC_POSTGRES_DB}" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)